pkgs/development/python-modules/ruamel_yaml_clib/default.nix

```diff
@@ -10,9 +10,8 @@ buildPythonPackage rec {
   pname = "ruamel.yaml.clib";
   version = "0.2.0";
 
-  src = fetchFromBitbucket {
-    owner = "ruamel";
-    repo = "yaml.clib";
+  src = fetchhg {
+    url = "http://hg.code.sf.net/p/ruamel-yaml-clib/code";
     rev = version;
     sha256 = "0kq6zi96qlm72lzj90fc2rfk6nm5kqhk6qxdl8wl9s3a42b0v6wl";
   };
```

Comment:
```
nix-build -A python3Packages.ruamel_yaml_clib.src --check
```
LGTM, at least on this front.  I'm surprised the hash didn't change between moving fetchers ...

BTW, even though the `src` input has the same directory hash, the reason this PR is a rebuild-the-world is that we have a dependency on the name of the directory for recursive hashes .... e.g., on master:

```
# On master
λ brh nixpkgs →  nix-build -A python3Packages.ruamel_yaml_clib.src
/nix/store/billhyciaqc7x2x3zhp0xyxld45pbpd8-source
λ brh nixpkgs →  nix-hash --type sha256 --base32 /nix/store/billhyciaqc7x2x3zhp0xyxld45pbpd8-source
0kq6zi96qlm72lzj90fc2rfk6nm5kqhk6qxdl8wl9s3a42b0v6wl

# On this PR:
λ brh nixpkgs →  nix-build -A python3Packages.ruamel_yaml_clib.src
/nix/store/h9k227wsfwd22bl7qwc8j724ajmxfnfs-hg-archive
λ brh nixpkgs →  nix-hash --type sha256 --base32 /nix/store/h9k227wsfwd22bl7qwc8j724ajmxfnfs-hg-archive
0kq6zi96qlm72lzj90fc2rfk6nm5kqhk6qxdl8wl9s3a42b0v6wl
```
Same hash . . . but the unpacked directory is named `hg-archive` instead of `source` :(